packages/server/graphql/mutations/helpers/notifications/NotificationHelper.ts

```diff
@@ -0,0 +1,96 @@
+import {IntegrationProviderMattermost} from '../../../../postgres/queries/getIntegrationProvidersByIds'
+import Meeting from '../../../../database/types/Meeting'
+import {SlackNotificationEvent} from '../../../../database/types/SlackNotification'
+import {DataLoaderWorker} from '../../../graphql'
+import {MattermostNotificationHelper} from './MattermostNotificationHelper'
+import {SlackNotificationHelper} from './SlackNotificationHelper'
+
+async function getSlack(
+  dataLoader: DataLoaderWorker,
+  event: SlackNotificationEvent,
+  teamId: string
+) {
+  const notifications = await dataLoader
+    .get('slackNotificationsByTeamIdAndEvent')
+    .load({event, teamId})
+  return notifications.map(SlackNotificationHelper)
+}
+
+async function getMattermost(dataLoader: DataLoaderWorker, meeting: Meeting, teamId: string) {
+  const {facilitatorUserId} = meeting
+  const provider = await dataLoader
+    .get('bestTeamIntegrationProviders')
+    .load({service: 'mattermost', teamId, userId: facilitatorUserId})
+  return provider ? [MattermostNotificationHelper(provider as IntegrationProviderMattermost)] : []
```

Comment:
Do you know why we return the types for our primaryKeyLoaderMakers but not for our other dataLoaders?

Reply:
We return a union type for this one so we need to cast to the correct one

https://github.com/ParabolInc/parabol/blob/eb295331feba7e2534c48012d252c2a52b802905/packages/server/postgres/queries/getIntegrationProvidersByIds.ts#L44-L47